Homework-№4-from-28.09.2022/task3.c

```diff
@@ -0,0 +1,205 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+#define maxSizeOfEntry 101
+
+typedef struct entry {
+    char phoneNumber[maxSizeOfEntry];
+    char name[maxSizeOfEntry];
+} dataBase;
+
+void instructionPrint() {
+    printf("This program has these commands:\n");
+    printf("0 - exit\n");
+    printf("1 - add an entry (name and phone number)\n");
+    printf("2 - print all available records\n");
+    printf("3 - find phone by name\n");
+    printf("4 - find name by phone\n");
+    printf("5 - save the current data to a file\n");
+}
+
+int readCommand() {
+    printf("Please, enter the command:");
+    int command = 0;
+    scanf("%d", &command);
+    while (!((command < 6) && (command > -1))) {
```

Comment:
```suggestion
    while (command < 0 || command > 5) {
```
Да поможет закон Де Моргана упростить код